src/index.js

```diff
@@ -2110,10 +2110,10 @@ function cov2ap(options = {}) {
     if (url.query.search) {
       let search = url.query.search;
       if (quoteSearch) {
-        search = `"${search.replace(/"/g, `\\"`)}"`;
+        search = `"${search.replace(/\\/g, "\\\\").replace(/"/g, `\\"`)}"`;
       } else {
         if (!/^".*"$/s.test(search) && search.includes('"')) {
```

Comment:
The if needs to be split, as it checks on the existence of a quote, to escape quotes. Back-slashes need to be escaped independent of that I think.

Reply:
That's the thing we weren't sure about. Should there be escape backslashes in every case? What if the user already escaped them already? To be 100 precise I think we would need to handle many more possibilities. 
That's what I pulled from the tokenizer from okra:
```js
// A search word is a sequence of one or more non-whitespace characters, excluding
// parentheses, double-quotes, and semicolons. It must not start with a single quote.
const WORD_REGEXP = new RegExp('^[^\\s()";\'][^\\s()";]*', 'u')
// A search phrase is a doublequoted string with backslash-escaped backslashes and doublequotes.
const PHRASE_REGEXP = new RegExp('^"(?:[^\\\\"]|\\\\[\\\\"])*"')
```

Reply:
Do we need to escape at all, if the search phrase is not quoted? Have you tried? I guess if it‘s not quoted it‘s also not escaped…

Reply:
I don't think so. That's what we currently do. We only escape backslashes if the string is quoted. The second if branch is also quoting in some circumstances.

Reply:
I mean the second if branch… there is checked that there are not quotes around, so every inner-quote needs to be escaped

Reply:
@oklemenz2 `search=foo\bar` is valid, but since you start quoting with the "searchQuote" option for strings that are not quoted the inner backslash becomes invalid, because \b is not a valid escape. That's the easy case we want to solve here. 

The other cases can be more complex. I would assume that `search=foo\bar "foo\"bar"` is also valid. This would probably be not valid anymore after the current conversions.

Reply:
Yes quoteSearch is a adapter configuration, that the developer can decide. So the fix for first if Block is fine I think. 

I have concerns for the else block, if the search is not quoted, thus no search phrase. I need to check myself some cases..